pyproject.toml

```diff
@@ -14,7 +14,20 @@ classifiers = [
 ]
 dependencies = [
     # Put the project's production dependencies here:
-
+    "allure-pytest==2.13.5",
```

Comment:
Only production dependencies should go here (of which we have none).

Currently, Python has no standard way to define development dependencies. This concept is tool-specific.

This repo uses the requirements file (this is the best option when using raw pip IMO).  So, we should leave the dev deps in the requirement file instead of putting them in pyproject.toml.

[This discussion](https://discuss.python.org/t/development-dependencies-in-pyproject-toml/26149) covers dev dependencies in great detail.